susan/apps/dhcp/pack_option.py

```diff
@@ -0,0 +1,86 @@
+# Copyright 2017 <thenakliman@gmail.com>
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+
+# Their are more than 70 options currently, two category of options are
+# important therefore included, more options can be included as per
+# requirement
+
+import copy
+import struct
+
+from susan.apps.dhcp import constants as const
+from susan.common import utils
+
+
+def struct_pack(value, **kwargs):
+    frmt = kwargs.get('format', '!%ss' % max(len(value), 2))
+    return struct.pack(frmt, value)
+
+
+def pack_list_address(values, **kwargs):
+    packed_ips = []
+    for value in values:
+        packed_ips.append(pack_ip(value))
+
+    return packed_ips
+
+
+def pack_ip(value, **kwargs):
+    return utils.get_packed_address(value)
+
+pack_list_address = pack_ip
```

Comment:
ok, got it, but I guess it can be generalized to one function?
def pack_address(values, **kwargs):
 if not isinstance(values, list):
    return utils.packed
  packed_ips = []
  for value in values:
       packed_ips.append(pack_ip(value))
   return packed_ips